coalib/parsing/filters/LanguageFilter.py

```diff
@@ -1,7 +1,3 @@
-from coalib.parsing.filters.Filter import filter
-
-
-@filter
 def language_filter(bear, args):
     """
     Filters the bears by ``LANGUAGES``.
```

Comment:
Something wrong here. I think you passed variable `bear` a tuple, but it shouldn't be a tuple.
`AttributeError: 'tuple' object has no attribute 'LANGUAGES'`